src/current/_includes/releases/v25.1/backward-incompatible.md

```diff
@@ -1,10 +1,33 @@
-Before [upgrading to CockroachDB v24.3]({% link v24.3/upgrade-cockroach-version.md %}), be sure to review the following backward-incompatible changes, as well as [key cluster setting changes](#v24-3-0-cluster-settings), and adjust your deployment as necessary.
+Before [upgrading to CockroachDB v25.1]({% link v25.1/upgrade-cockroach-version.md %}), be sure to review the following backward-incompatible changes, as well as [key cluster setting changes](#v25-1-0-cluster-settings), and adjust your deployment as necessary.
 
-If you plan to upgrade to v24.3 directly from v24.1 and skip v24.2, be sure to also review the [v24.2 release notes]({% link releases/v24.2.md %}) for backward-incompatible changes from v24.1.
+- The old `BACKUP TO`, `RESTORE FROM <collectionURI>`, and `SHOW BACKUP IN <collectionURI>` syntaxes are now fully deprecated and no longer usable. [#133610][#133610]
 
-- Upgrading to v24.3 is blocked if no [license]({% link v24.3/licensing-faqs.md %}) is installed, or if a trial/free license is installed with telemetry disabled. [#130576][#130576]
+- Altering a paused backup schedule's recurrence or location no longer resumes the schedule. [#134829][#134829]
 
-[#130576]: https://github.com/cockroachdb/cockroach/pull/130576
+- `BACKUP`/`RESTORE` statements no longer return index entries and bytes backed up/restored. [#134516][#134516]
 
-{% comment %}Remove this anchor when it is added to the v24.3.0 GA release notes{% endcomment %}
-<a id="v24-3-0-cluster-settings"></a>
+- Updated the default for `legacy_varchar_typing` session setting. This setting reverts the changes of [#133037](https://github.com/cockroachdb/cockroach/pull/133037) that causes the change in typing behavior described in [#137837](https://github.com/cockroachdb/cockroach/pull/137837). Specifically, it makes type-checking and overload resolution ignore the newly added "unpreferred" overloads. As of v25.1.0 this setting defaults to `off`. [#137844][#137844]
```

Comment:
Suggest using the actual session var description instead. I think the user impact is clearer.

```suggestion
- Introduced the `legacy_varchar_typing` session setting. If `on`, type checking and overload resolution for VARCHAR types ignore overloads that cause errors, allowing comparisons between VARCHAR and non-STRING-like placeholder values to execute successfully. If `off`, type checking of these comparisons is more strict and must be handled with explicit type casts. As of v25.1.0 this setting defaults to `off`. [#137844][#137844]
```